src/wui/buildingwindow.cc

```diff
@@ -277,6 +277,24 @@ void BuildingWindow::create_capsbuttons(UI::Box* capsbuttons, Widelands::Buildin
 			stopbtn->sigclicked.connect([this]() { act_start_stop(); });
 			capsbuttons->add(stopbtn);
 
+			if (productionsite->descr().is_infinite_production_useful() ||
+			    productionsite->infinite_production()) {
```

Comment:
can we have infinite_production whithout is_infinite_production useful or with only 1 of them present?